### DISCUSSION TITLE
Stopping a timer doesn't remember timer start

### DISCUSSION THREAD

**Discussion by voss** [babybuddy/babybuddy#350] [Support] [answered]
I just updated to 1.9.1. following @cdubz very helpful assistance [over here](https://github.com/babybuddy/babybuddy/discussions/349). I really like the additions since 1.5.1. which I came from.

I had to roll back pretty quickly though, because as I wanted to stop a running timer (it was the same situation for feed, sleep, or tummy time timer), the "start time" field was blank on the submit page, where it would normally have time the timer was started.

I stupidly didn't grab a screenshot of it, and I rolled back to 1.5.1. to not make the night tracking more challenging than necessary (: 

I wanted to hear if anyone else has seen this in 1.9.1? I will try to see if I can reproduce it. If I can, what would be some helpful data points to note down for tracking down the root cause?

**Comment by cdubz** [comment, marked as answer]
This _might_ be related to https://github.com/babybuddy/babybuddy/issues/337. You could give v1.9.0 a try and if you can't reproduce the issue there it probably is #337. I have some commits on `master` also that _may_ address this issue if you want to try living on the edge (if only briefly) to test there...

**Reply by voss** [reply]
Sorry for not coming back to this earlier. I chickened out of trying the master and just went with 1.9.0. in the end, where it worked.
